privacy-policy

```diff
@@ -0,0 +1,76 @@
+Nuts respecteert de privacy van de bezoekers van haar website.
+Nuts geeft de door u verstrekte persoonsgegevens niet door aan andere partijen, tenzij u hier nadrukkelijk toestemming voor hebt gegeven of hiertoe een verplichting bestaat op basis van wet- of regelgeving.
+
+## Begrippen
```

Comment:
Kunnen we beginnen met wat we wel en niet doen en verwijzen naar uitleg vd begrippen onderin het document?